xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp

```diff
@@ -113,12 +112,12 @@ void CEngineStats::UpdateStream(CActiveAEStream *stream)
   {
     if (str.m_streamId == stream->m_id)
     {
-      float delay = 0;
+      double delay = 0;
       str.m_syncState = stream->m_syncState;
       str.m_syncError = stream->m_syncError.GetLastError(str.m_errorTime);
       if (stream->m_processingBuffers)
       {
-        str.m_resampleRatio = stream->m_processingBuffers->GetRR();
+        str.m_resampleRatio = (double)stream->m_processingBuffers->m_inputFormat.m_sampleRate / stream->m_processingBuffers->m_outputFormat.m_sampleRate;
```

Comment:
Use direct the `static_cast` for things like this, is requested now all new Kodi parts
.